File: backend/crates/akashic-extraction/tests/fixtures/dart/05-nested.dart
int outer(List<int> items) {
  const factor = 2;
  int scale(int x) {
    return x * factor;
  }
  int total() {
    return items.map(scale).reduce((a, b) => a + b);
  }
  return total();
}

int standalone() {
  return outer([1, 2, 3]);
}

File: backend/crates/akashic-extraction/tests/fixtures/swift/05-nested.swift
func outer(_ items: [Int]) -> Int {
    let factor = 2
    func scale(_ x: Int) -> Int {
        return x * factor
    }
    func total() -> Int {
        return items.map(scale).reduce(0, +)
    }
    return total()
}

func standalone() -> Int {
    return outer([1, 2, 3])
}
